go.mod

```diff
@@ -45,6 +45,7 @@ require (
 	github.com/google/go-cmp v0.5.9
 	github.com/google/uuid v1.3.0
 	github.com/gosuri/uilive v0.0.4
+	github.com/hashicorp/go-retryablehttp v0.7.5
```

Comment:
We can run `go mod tidy` just in case.

Reply:
I'll rerun again before merging to make sure this is up to date